doc/github_actions.rst

```diff
@@ -0,0 +1,39 @@
+Integration with GitHub Actions
+===============================
+
+`Universum` requires no special integration with `GitHub Actions <https://docs.github.com/en/actions>`_. It is usually
+launched as one long step in a single build stage.
+
+.. warning::
+
+    GitHub Actions doesn't support `report_artifacts` key in configuration. If specified, a broken link will be created.
```

Comment:
'GitHub Actions' is plural, so _Actions don't support_

Reply:
I disagree with you. 'GitHub Actions' is the name of platform, so it's like a [The Times](https://en.wikipedia.org/wiki/The_Times) - singular.

Reply:
Then please use the quotes to avoid misunderstanding.